README.md

```diff
@@ -175,43 +180,58 @@ $ chamber export [--format <format>] [--output-file <file>]  <service...>
 `export` provides ability to export secrets in various file formats. The following
 file formats are supported:
 
-* json (default)
-* yaml
-* java-properties
-* csv
-* tsv
-* dotenv
-* tfvars
+- json (default)
+- yaml
+- java-properties
+- csv
+- tsv
+- dotenv
+- tfvars
 
 File is written to standard output by default but you may specify an output
 file.
 
-To set env vars in your terminal you can use the `chamber env` command. For example, 
+To set env vars in your terminal you can use the `chamber env` command. For example,
+
 ```shell
 source <(chamber env service)`
 printf "%s" "$SERVICE_VAR"
 ```
 
 ### Importing
+
 ```bash
-$ chamber import <service> <filepath>
+$ chamber import [--normalize-keys] <service> <filepath>
 ```
 
 `import` provides the ability to import secrets from a json or yaml file (like the kind
 you get from `chamber export`).
 
+<!-- prettier-ignore -->
+> __Note__
+> By default, `import` will **not** normalize key inputs, meaning that keys will
+> be written to the secrets backend in the format they exist in the source file.
+> In order to normalize keys on import, provide the `--normalize-keys` flag
```

Comment:
I'd suggest adding an example of normalized vs. "original" keys and illustrating what actually changes.

Reply:
Sounds good! I was relying on tests for this, but will add an example 👍

Reply:
Added in https://github.com/segmentio/chamber/pull/370/commits/6174a82a5fca3cee236162b365d894dcab3c7014